React/src/DataCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+function ResourceCell(props: any): JSX.Element {
+  function isWeekEnd(date: Date): boolean {
+    const day = date.getDay();
+    return day === 0 || day === 6;
+  }
+
+  function getCurrentTraining(date: number, employeeID: number): string {
+    const result = (date + employeeID) % 3;
+    const currentTraining = `training-background-${result}`;
+
+    return currentTraining;
```

Comment:
Let's make it shorter:

```suggestion
    return `training-background-${result}`;
```